python/race_model/EloRaceModel.py

```diff
@@ -91,7 +103,8 @@ def generateModel(self):
                 if raceId in self.raceResultsData:
                     resultsForRace = self.raceResultsData[raceId]
                     self._addNewDriversAndConstructors(resultsForRace, year)
-                    self._addNewTrack(data.circuitId)
+                    # self._addNewTrack(data.circuitId)
+                    self._addNewTracksToEntities(data.trackId)
```

Comment:
This seems like a good place to add the call to `addNewTracksToEntities`, well done. But the call to _addNewTrack should not be commented out, we still need it.